prog/weaver/reclaim.go

```diff
@@ -0,0 +1,54 @@
+package main
```

Comment:
I added logging of the addresses found.
Probably this should be at debug level, but I left it at INFO for now.